File: app/src/main/java/de/werkelmann/rekote/view/dialogs/InfoExceptionDialog.java
package de.werkelmann.rekote.view.dialogs;

import android.app.AlertDialog;
import android.app.Dialog;
import android.content.DialogInterface;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.v4.app.DialogFragment;

import de.werkelmann.rekote.R;

public class InfoExceptionDialog extends DialogFragment {

    @Override
    @NonNull
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        AlertDialog.Builder builder = new AlertDialog.Builder(getContext());

        builder.setMessage(R.string.error_no_info)
                .setTitle(R.string.info)
                .setPositiveButton(R.string.ok, new DialogInterface.OnClickListener() {
                    @Override
                    public void onClick(DialogInterface dialog, int which) {
                        dialog.dismiss();
                    }
                });

        return builder.create();
    }
}

File: app/src/main/kotlin/de/werkelmann/rekote/view/dialogs/InfoExceptionDialog.kt
package de.werkelmann.rekote.view.dialogs

import android.app.AlertDialog
import android.app.Dialog
import android.os.Bundle
import android.support.v4.app.DialogFragment
import de.werkelmann.rekote.R

class InfoExceptionDialog : DialogFragment() {

    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        val builder = AlertDialog.Builder(context)

        builder.setMessage(R.string.error_no_info)
                .setTitle(R.string.info)
                .setPositiveButton(R.string.ok) { dialog, _ -> dialog.dismiss() }

        return builder.create()
    }
}
